src/components/CardMenus/CommonCard.js

```diff
@@ -40,39 +45,82 @@ const useStyles = makeStyles((theme) => ({
   expandOpen: {
     transform: 'rotate(180deg)',
   },
-  avatar: {
-    backgroundColor: 'red',
-  },
+
   image: {
     padding: '30px',
-    border: '10px',
-    borderRadius: '200px',
+    border: '100px',
+    boxShadow: '10px 10px F0F0F0',
+  },
+  drawer: {
+    width: drawerWidth,
+    flexShrink: 0,
+  },
+  drawerPaper: {
+    width: drawerWidth,
+  },
+  Price: {
+    paddingTop: '10px',
+    paddingLeft: '50px',
+    fontSize: '18px',
+    fontWeight: '700',
+  },
+
+  btn: {
+    marginTop: '10px',
+    minWidth: '10px',
   },
 }));
-const CommonCard = ({ name, price, resturantName, img }) => {
+const CommonCard = ({ name, price, resturantName, img, buttonText }) => {
   const classes = useStyles();
-
+  const { root, content, header, image, Price, btn } = classes;
+  const [showButton, setShowButton] = useState(false);
   return (
-    <div>
-      <Card className={classes.root}>
-        <Box className={classes.image}>
-          <CardMedia className={classes.media} image={img} title={name} />
-        </Box>
+    <Grid item md={3}>
+      <Card className={root}>
+        <div className={image}>
+          <div style={{ boxShadow: '10px 10px 13px lightgrey' }}>
+            <CardMedia className={classes.media} image={img} title={name} />
+          </div>
+        </div>
         <CardHeader
-          className={classes.header}
-          // eslint-disable-next-line react/jsx-no-duplicate-props
+          className={header}
           subheader={<h4>{resturantName}</h4>}
-          title={<h2>{name}</h2>}
+          title={
+            <Typography className={classes.foodTitle} variant="h2">
+              {name}
+            </Typography>
+          }
         />
-        <CardContent className={classes.content}>
-          <Typography color="textSecondary" component="p" variant="h4">
-            <b>
-              Price:<span style={{ marginLeft: '10px', marginBottom: '20px' }}>{price}</span>
-            </b>
-          </Typography>
+        <CardContent className={content}>
+          <div style={{ marginTop: '20px', display: 'flex' }}>
+            <div className={btn}>
+              {showButton ? (
+                <div>
+                  <CommonButton color="secondary" minwidth="10px">
+                    +
+                  </CommonButton>
+                  {'  '}
+                  <span style={{ margin: '5px 5px' }}>1</span>
+                  {'  '}
+                  <CommonButton color="secondary" minwidth="15px">
+                    -
+                  </CommonButton>
+                </div>
+              ) : (
+                <div>
+                  <CommonButton color="secondary" onClick={() => setShowButton(true)} style={{ width: '50px' }}>
```

Comment:
remove all inline styling and why do we need to show the quantity buttons on clicking Add to card button ??
Is @mirzasahaab has told you to do this then it is ok .. but if not, then there should be buttons directly for adding the food to the cart .. and "-" button should be disabled if the quantity is 0 ..